spec/dummy/db/schema.rb

```diff
@@ -183,6 +189,7 @@
     t.boolean  "address_finder_error",   default: false
     t.datetime "created_at",                             null: false
     t.datetime "updated_at",                             null: false
+    t.string   "type"
```

Comment:
Shall we add some validations to this? Also a `null: false` feels like is needed. This looks like 1 step in the Single Table Inheritance direction, hence the `type` attribute is going to play a fundamental role going forward :)

Reply:
Added a `null: false` in https://github.com/DEFRA/waste-exemptions-engine/pull/138/commits/34ecbe11f8c572198c348a054ea472ad4e5a26bc

Reply:
Oops, broke everything. Ignore that for now...

Reply:
Required a few other commits first to refactor, but think I've sorted this now: https://github.com/DEFRA/waste-exemptions-engine/pull/138/commits/938be38d708a451ee4f956808e2dfeb6367a081d